core/templates/dev/head/css/oppia.css

```diff
@@ -875,7 +875,8 @@ textarea {
 .dropdown-menu {
   padding: 5px 0;
   margin: 2px 0 0;
-  max-width: 220px;
+  margin-left: 10px;
```

Comment:
This change will affect dropdown menus throughout the site. I do not think this is the right way to do it. Can you limit the scope only for category selection in mobile view?

Reply:
I did some changes